shell/src/main/java/org/apache/accumulo/shell/commands/InsertCommand.java

```diff
@@ -171,6 +180,11 @@ public Options getOptions() {
         "durability to use for insert, should be one of \"none\" \"log\" \"flush\" or \"sync\"");
     o.addOption(durabilityOption);
 
+    tableNameOption =
+        new Option("", "tablename", true, "name of table in which data is being inserted");
+    tableNameOption.setArgName("tablename");
+    o.addOption(tableNameOption);
+
```

Comment:
What's the reason for not supporting `-t` short name?

Reply:
I chose not to use the short form to minimize any confusion with the <code>timestamp</code> parameter (-ts). There is also a <code>timeout<code> parameter that does not off a shortened name. If you think it would be nice to include one, I have no problem adding one. Could always use '-n' for 'name' perhaps. Let me know your thoughts.

Reply:
If there's not already a `-t`, I don't think it's a problem. I don't think there's any concern over conflicting with `-ts`. In fact, most commands, use the standard `OptUtil.tableOpt`. See  https://github.com/apache/accumulo/blob/main/shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java#L93 and https://github.com/apache/accumulo/blob/main/shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java#L395.
I would recommend doing the same here, if there's no existing `-t`.